politeiawww/cmsuser.go

```diff
@@ -546,6 +546,97 @@ func (p *politeiawww) getCMSUserByIDRaw(id string) (*user.CMSUser, error) {
 	return ubir.User, nil
 }
 
+func (p *politeiawww) getCMSUserWeights() (map[string]int64, error) {
+	userWeights := make(map[string]int64, 1080)
+
+	/*
+		1) Determine most recent payout month
```

Comment:
Its unclear from this description that the user weight is equal to the total number of minutes billed over the past 6 months.